spec/lib/mdot/client_spec.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe MDOT::Client do
+  subject { described_class.new(user) }
+
+  let(:user_details) do
+    {
+      first_name: 'Greg',
+      last_name: 'Anderson',
+      middle_name: 'A',
+      birth_date: '1949-03-04',
+      ssn: '000555555'
+    }
+  end
+
+  let(:user) { build(:user, :loa3, user_details) }
+
+  describe '#get_supplies' do
+    context 'with a valid supplies response' do
+      it 'returns an array of supplies' do
```

Comment:
Should this say "returns an array"?  The test seems to only test if it's an `MDOT::Response`, I think the name of the spec should match up with what it tests.